.travis.yml

```diff
@@ -11,3 +11,11 @@ branches:
 cache:
   directories:
     - node_modules
+after_success:
+  - if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+    docker build -t cerner/kaiju-rails:latest ./rails;
+    docker build -t cerner/kaiju-node:latest ./node;
```

Comment:
Do we want to go ahead and tag these as 0.1.0.SNAPSHOT? Or wait until we're ready to tag at 0.1.0 ?